packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -146,6 +146,13 @@ class DriveCommand extends RunCommandBase {
     if (argResults['use-existing-app'] == null) {
       globals.printStatus('Starting application: $targetFile');
 
+      if (isWebPlatform) {
+        throwToolExit(
+            'Flutter Driver (web) does not support running without use-existing-app.\n'
```

Comment:
nit: 2 space indent on a continued line. Here and below.

Reply:
Fixed.